src/index.html

```diff
@@ -149,6 +149,61 @@ <h2>Overview</h2>
 				</p>
 			</div>
 		</section>
+		<section class="organizers-section">
+			<div class="container">
+				<h1 class="organizers__header">Organizers</h1>
+				<h3>Click on a profile to learn more.</h3>
+				<div class="organizers__profiles row">
+					<figure
+						tabindex="0"
+						class="profile profile--selected col"
```

Comment:
I may be missing it, but is the `col` class needed (here and below)?

Reply:
Didn't use col classes in CSS. Removed.